### ISSUE TITLE
Hawkeye Image with v1.1.1 on Public Registry

### ISSUE THREAD

**Issue by akr24** [Stono/hawkeye#73]
### Description
I'll pull stono/hawkeye from the public docker registry and try to run a scan, but the scan v1.0.3 will run instead of an updated v1.1.1

### Steps to Reproduce
docker pull stono/hawkeye
docker run --rm -v $PWD:/target stono/hawkeye

**Expected behavior:** [What you expect to happen]
`Welcome to Hawkeye v1.1.1!`
then scan following

**Actual behavior:** [What actually happens]
`Welcome to Hawkeye v1.0.3!`
then scan and eventual pass/failure

### Version
^ see above

### Additional Information
Basically, has the docker image been updated? If so, are there any steps I'm missing in trying to run a scan?

**Comment by felixhammerl** [comment]
confirmed. working on it.

**Comment by felixhammerl** [comment]
See: https://travis-ci.org/Stono/hawkeye

```
The command '/bin/sh -c pip install safety==1.6.1 piprot==0.9.8 bandit==1.4.0' returned a non-zero code: 1
```
Looks like the rest of the installation does not go through, because ... you know ... pypi sucks.
